data/mods/More_Survival_Tools/recipes.json

```diff
@@ -0,0 +1,1003 @@
+[
```

Comment:
Thank you Kevin, if I recall correctly, for pointing out you can set recipes to override existing ones.